Formula/inko.rb

```diff
@@ -18,13 +18,16 @@ class Inko < Formula
   end
 
   depends_on "coreutils" => :build
+  depends_on "llvm@15" => :build
   depends_on "rust" => :build
+  depends_on "zstd"
 
   uses_from_macos "libffi", since: :catalina
   uses_from_macos "ruby", since: :sierra
 
   def install
-    system "make", "build", "PREFIX=#{prefix}", "FEATURES=libffi/system"
+    ENV.prepend_path "PATH", Formula["coreutils"].opt_libexec/"gnubin"
```

Comment:
This is because the inko makefile uses `install -D`, and `-D` is a gnu install extension that is not available on the macos BSD install. I copied this from another formula, so hopefully it's close to idiomatic.